src/redefined.resolver.ts

```diff
@@ -28,7 +28,7 @@ export class RedefinedResolver implements Resolver {
         // throwErrorOnInvalidDomain - non-rewritable parameter
         // If it is true, then everything will fall on an error in one resolver
         return flatten(
-          await Promise.all(this.resolvers.map(it => it.resolve(domain, { ...customOptions, throwErrorOnInvalidDomain: false }, networks)))
+          await Promise.all(this.resolvers.map(it => it.resolve(domain.toLowerCase(), { ...customOptions, throwErrorOnInvalidDomain: false }, networks)))
```

Comment:
I think this fix is not needed fot JS lib
Only backend and connect-web
But it needs to be clarified

Reply:
why not? but I believe lowercase should happen in RedefinedEmailResolverService and RedefinedNicknameResolverService

Reply:
@rnovokhatsky 
May be only in these services
BUT is it correct for ENS and Uns?

Reply:
ENS and Uns encapsulate their own logic, we should not bother

Reply:
@au-e2xlabs 
So, need to transfer this code to `redefined-resolver.service.ts`